lte/gateway/python/integ_tests/s1aptests/s1ap_wrapper.py

```diff
@@ -203,7 +203,7 @@ def configIpBlock(self):
         print("************************* Waiting for IP changes to propagate")
         self._mobility_util.wait_for_changes()
 
-    def configUEDevice(self, num_ues, reqData=[]):
+    def configUEDevice(self, num_ues, reqData=[], static_ips=[]):
```

Comment:
Not related to your changes but better to fix now, can you please remove the default mutable parameters being passed here to instead pass `None`? then inside the function we can do

```
if reqData is None:
    reqData = []
```